contribs/gmf/less/mobile.less

```diff
@@ -122,12 +125,24 @@ span.twitter-typeahead {
       content: '\f00d';
       color: grey;
       font-family: FontAwesome;
-      font-size: 1.5em;
+      font-size: 2em;
```

Comment:
OK for me, as long as the size feeling is coherent with the rest (+ icon centered vertically please)